Makefile

```diff
@@ -47,8 +47,7 @@ protobuf:
 # Golangci-lint
 .PHONY: golangci-lint
 golangci-lint:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.58.1
-	golangci-lint run --timeout 60m ./...
+	go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.58.2 run --timeout 60m --new-from-rev dd5f9afe8948186c76fe6b8b1193d7a8f46919d8 ./...
```

Comment:
I ran into this locally the other day (that make still ran my system installed version), so thanks for this.